### ISSUE TITLE
Carousel Produces Accessibility Violations

### ISSUE THREAD

**Issue by neilchaudhuri** [OwlCarousel2/OwlCarousel2#2045]
The accessibility testing tool [Axe](https://www.deque.com/products/axe/), which is part of Google's [Lighthouse](https://developers.google.com/web/tools/lighthouse/) testing tool, considers the absence of an "orphan" `li` element to be a "critical" violation. There are two ways to solve this:

- Each `li` has an immediate `ol` or `ul` parent.
- Each `li` has a parent with `role="list"`.

The latter should work for the `div` elements with `owl-item` class. This relatively simple change would make things much better for disabled users.

Thanks.
